gradle.properties-example

```diff
@@ -41,6 +41,7 @@ wp.e2e.wp_com_user.email=e2eflowtestingmobile@example.com
 wp.e2e.wp_com_user.password=mocked_password
 wp.e2e.wp_com_user.site_address=e2eflowtestingmobile.wordpress.com
 wp.e2e.wp_com_user.username=e2eflowtestingmobile
+wp.e2e.wp_com_passwordless_user.email=e2eflowtestingmobile+passwordless@example.com
```

Comment:
Unrelated question: How do we inform devs when changes are made in the properties example files like here – as I'm guessing they are expected to transfer those additions/changes to their local `.properties` files on their machine?